android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -317,6 +317,7 @@ public String getInstallReferrer() {
             getCurrentActivity().checkCallingOrSelfPermission("android.permission.READ_PHONE_NUMBERS") == PackageManager.PERMISSION_GRANTED)) {
       TelephonyManager telMgr = (TelephonyManager) this.reactContext.getApplicationContext().getSystemService(Context.TELEPHONY_SERVICE);
       constants.put("phoneNumber", telMgr.getLine1Number());
+      constants.put("imeiNumber", telMgr.getDeviceId());
```

Comment:
I looked into this - it needs more changes than this. There may be 0 or more (up to 3 currently) "phones" on Android, so you need to first get the count: https://developer.android.com/reference/android/telephony/TelephonyManager.html#getPhoneCount()

But that's only available for API23+, so you have to check if it's API23+ and if not to get that info.

Then from 0 to phone count, you need to getPhoneType, then either getMeid or getImei for the slot (for API26+) or getDeviceId for API25- 

I would suggest an API name like getCarrierSubscriberIDs() that returns a string[], containing 0 or more entries, each possibly null, populated like the above.

As a side benefit, people can tell how many cell slots a phone has by doing .length on the return value

I recognize that's a pretty significant change request but I think it's the right thing to do - the results don't necessarily make sense on a lot of devices (including mine, which has one SIM in slot 2) otherwise. I'll approve this if it works like that